package.json

```diff
@@ -21,11 +27,12 @@
 		"typescript": "^5.0.4"
 	},
 	"engines": {
-		"node": "^18.13.0"
+		"node": ">=18.13.0"
 	},
 	"packageManager": "pnpm@8.3.1",
 	"private": true,
 	"scripts": {
+		"prepare": "npm run src:build",
```

Comment:
Why use `npm` here?

Reply:
To make it installable with NPM. If your project uses NPM, and have no idea about pnpm, `pnpm src:build` call will fail.